public/static/locales/en/vice-admin.json

```diff
@@ -1,4 +1,10 @@
 {
+    "accessRequests": "Access Requests",
+    "showAllRequests": "Show All Requests",
+    "accessRequestApprovedMsg": "Your request to access VICE apps is approved.",
+    "accessRequestDeniedMsg": "We are unable approve your request to access VICE apps at this moment.",
```

Comment:
I was going to comment on how this message might be a little misleading, but I don't think it's actually used anywhere? 🤔

Reply:
Good catch. I added a feature of admins to add a custom instead of this one.